### ISSUE TITLE
python crashed when call argmax

### ISSUE THREAD

**Issue by haolu23** [tensorly/tensorly#44]
```python
import tensorly as tl
import numpy as np

# default backend is mxnet
x = np.random.random(100)

# crashed here
tl.tensor(x).argmax()
# or
np.argmax(tl.tensor(x))
```
My numpy version 1.12.1, tensorly version 0.12.1

**Comment by JeanKossaifi** [comment]
The latest version to TensorLy is 0.3.0 and there is not yet a 0.12: you probably meant an earlier version? What error/traceback do you get?

Just looking at the code it seems to me that you have a version of TensorLy that still used mxnet as the default backend. If that is the case, you are using numpy functions on an MXNet NDArray, and the way you use argmax is incorrect (you have to specify the axis in MXNet when using argmax as it doesn't support global reduction). You should be able to fix that by simply calling `tl.set_backend('numpy')`, but I recommend you update to the latest version of TensorLy: `pip update -U tensorly` or `conda install -c tensorly tensorly`.

**Comment by haolu23** [comment]
You are right, it is 0.2.0 (updated my original post).  Python2.7 crashed without any error/traceback.  I tried Python3.5, same crash. 
passing in axis argument to argmax works fine.  Probably a MXNet problem. 
```python
np.argmax(tl.tensor(x), axis=0)
```
